src/webgl/material.js

```diff
@@ -1054,7 +1054,10 @@ p5.RendererGL.prototype._applyColorBlend = function(colors) {
 
   const isTexture = this.drawMode === constants.TEXTURE;
   const doBlend =
-    isTexture || colors[colors.length - 1] < 1.0 || this._isErasing;
+    isTexture ||
+    this.curBlendMode !== constants.BLEND ||
```

Comment:
This is a great and important change! I remember when we first added blendmodes to WebGL, we were getting a bunch of issues that all involved transparency or textures. It was easy to forget that people blend fully opaque colors as well 🤦‍♀️ . Great job finding a small improvement that makes a big difference!

Reply:
I dread every transparency/blending bug I encounter because it's so easy to mess up in WebGL 😅 I really appreciate your work setting all of it up so that transparency Just Works in p5!